vuu-ui/packages/vuu-layout/src/drag-drop/Draggable.ts

```diff
@@ -216,24 +216,24 @@ function dragMousemoveHandler(evt: MouseEvent) {
     _dragMoveCallback?.(newX, newY);
   }
 
-  if (_simpleDrag) {
+  if (_simpleDrag || !_dragContainer || !_validDropTargetPaths) {
```

Comment:
early exit in the case of !_validDropTargetPaths is a breaking change. It's normal for 
_validDropTargetPaths to not be defined. The use of a non-null assertion on the original code was wrong.